Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/hibernate/LoadTestDataServiceImpl.java

```diff
@@ -48,64 +54,182 @@
     private static final Logger LOG = LoggerFactory.getLogger(LoadTestDataServiceImpl.class);
     private PatientDAO patientDAO = PatientDAO.getPatientDAOInstance();
     private PersonnameDAO personnameDAO = PersonnameDAO.getPersonnameDAOInstance();
+    private IdentifierDAO identifierDAO = IdentifierDAO.getIdentifierDAOInstance();
+    private AddressDAO addressDAO = AddressDAO.getAddressDAOInstance();
+    private PhonenumberDAO phonenumberDAO = PhonenumberDAO.getPhonenumberDAOInstance();
 
     @Override
     public List<Patient> getAllPatients() {
-        LOG.info("Service-getAll-patients");
         return patientDAO.getAll();
     }
 
     @Override
     public boolean deletePatient(Patient patient) {
-        LOG.info("Service-delete-patient");
         return patientDAO.deleteTransaction(patient);
     }
 
     @Override
-    public Patient getPatientById(Long id) {
-        LOG.info("Service-getPatient-byID");
```

Comment:
Its better to have the log statements as it will help with debugging.  I modified these log statements as part of my PR#1482 to be able to get a clear picture of which implementation is being called.

Reply:
You probably have to accept my changes before merging. Its gonna result in conflict.